packages/neon/neon/lib/src/widgets/image.dart

```diff
@@ -328,13 +286,118 @@ class NeonApiImage extends StatelessWidget {
   }
 }
 
+/// Nextcloud URL image.
+///
+/// Extension for [NeonCachedImage] providing a [NextcloudClient] to the caller
+/// to retrieve the image from an URL.
```

Comment:
This is wrong as it does not provide a `NextcloudClient`.
Some copy :spaghetti: